src/main/cpp/agent.cpp

```diff
@@ -250,7 +251,7 @@ static bool RegisterJvmti(jvmtiEnv *jvmti) {
     return true;
 }
 
-static char *safe_copy_string(const char *value, const char *next) {
```

Comment:
I'm using it to allocate new `char *` in a couple of places: _agent.cpp_, _controller.cpp_, _control.cpp_ to make sure that all strings in `Configuration` were instantiated in the same way to free them within `Profiler` without knowing how memory was allocated: free/new.